### ISSUE TITLE
Fix false positives for interpolation and shorthand in font-family-name-quotes

### ISSUE THREAD

**Issue by bdpartridge** [stylelint/stylelint#4154]
I'm getting false positives related to a Sass interpolation in a `font` shorthand declaration value. The SCSS looks something like:

```scss
.foo {
  font: 12px/#{customFunc($some-length)} Roboto, sans-serif;
}
```

The unexpected errors are:

```
Expected quotes around "} Roboto" (font-family-name-quotes)
Expected "#{customFunc" to be "#{customfunc" (function-name-case)
```

And the related rules are configured as follows:

```json
{
  "rules": {
    "font-family-name-quotes": "always-where-recommended",
    "function-name-case": ["lower", {"ignoreFunctions": ["customFunc"]}]
  }
}
```

This is happening with stylelint 10.1.0, when running from CLI.

**Comment by evilebottnawi** [comment]
It is bug, feel free to send a PR, thanks for issue

**Comment by bdpartridge** [comment]
@evilebottnawi Thanks for confirming this is a bug. I'll try to find some time over the next several weeks to submit a PR.

**Comment by jeddy3** [comment]
Let's use the `isStandardSyntaxValue` util in the `font-family-name-quotes` rule to ignore values with interpolation.

(Ignore `function-name-case` as it's a stylistic rule.)

I've labelled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide.

**Comment by kimulaco** [comment]
@jeddy3 Hi. I would like to work on this issue and create a PR.

**Comment by jeddy3** [comment]
@kimulaco Thank you for offering to contribute. You're welcome to create a PR to fix the bug in the `font-family-name-quotes` rule.
